Makefile

```diff
@@ -190,7 +189,7 @@ splunk-redhat-8: base-redhat-8 ansible
 		-f splunk/common-files/Dockerfile \
 		--build-arg SPLUNK_BASE_IMAGE=base-redhat-8 \
 		--build-arg SPLUNK_BUILD_URL=${SPLUNK_LINUX_BUILD_URL} \
-		-t splunk-redhat-8:${IMAGE_VERSION} .
+		-t splunk-redhat-8-without-scloud:${IMAGE_VERSION} .
```

Comment:
Is this going to affect any downstream dependencies that were looking for "splunk-redhat-8"